nemo_curator/scripts/semdedup/extract_dedup_data.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import os
+from datetime import datetime
+
+from nemo_curator.log import create_logger
+from nemo_curator.modules.config import SemDedupConfig
+from nemo_curator.modules.semantic_dedup import SemanticClusterLevelDedup
+from nemo_curator.utils.distributed_utils import get_client
+from nemo_curator.utils.script_utils import ArgumentHelper
+
+
+def main():
+    semdedup_config = SemDedupConfig.from_yaml("config.yaml")
```

Comment:
Same comments as with compute_embeddings.py, move this to argparse, move stuff to `attach_args`, etc.

Reply:
https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/eaada91623f7b41d78276bc7ef7f04bc97c7acb0